apps/spark/src/main/java/com/linkedin/openhouse/jobs/util/TableStatsCollectorUtil.java

```diff
@@ -79,11 +115,38 @@ public static IcebergTableStats populateStatsOfAllReferencedFiles(
   public static IcebergTableStats populateStatsForSnapshots(
       String fqtn, Table table, SparkSession spark, IcebergTableStats stats) {
 
-    Dataset<Row> dataFiles = getAllDataFilesCount(table, spark, MetadataTableType.FILES);
-    long countOfDataFiles = dataFiles.count();
-
-    // Calculate sum of file sizes in bytes in above allDataFiles
-    long sumOfFileSizeBytes = getSumOfFileSizeBytes(dataFiles);
+    // TODO: turn these longs into attributes of a class and use the class as a schema which spark
```

Comment:
is this stale since you already have `DataFilesSummary` ?

Reply:
yep